app/models/user.rb

```diff
@@ -11,8 +11,18 @@ class User < ApplicationRecord
   validates :password, presence: true, 
     length: { minimum: Settings.validations.password.min_length }, allow_nil: true
   
-    has_secure_password
-    has_many :microposts, dependent: :destroy
+  has_secure_password
+  has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
+                                  foreign_key: "follower_id",
```

Comment:
change from `"Relationship"` to `Relationship.name`

change from `"followed_id"` to `:followed_id`